cache/lru_cache_test.cc

```diff
@@ -76,32 +77,33 @@ class LRUCacheTest : public testing::Test {
   void Erase(const std::string& key) { cache_->Erase(key, 0 /*hash*/); }
 
   void ValidateLRUList(std::vector<std::string> keys,
-                       size_t num_high_pri_pool_keys = 0) {
+                       size_t num_high_pri_pool_keys = 0,
+                       size_t num_low_pri_pool_keys = 0,
+                       size_t num_bottom_pri_pool_keys = 0) {
     LRUHandle* lru;
     LRUHandle* lru_low_pri;
-    cache_->TEST_GetLRUList(&lru, &lru_low_pri);
+    LRUHandle* lru_bottom_pri;
+    cache_->TEST_GetLRUList(&lru, &lru_low_pri, &lru_bottom_pri);
     LRUHandle* iter = lru;
-    bool in_high_pri_pool = false;
     size_t high_pri_pool_keys = 0;
-    if (iter == lru_low_pri) {
-      in_high_pri_pool = true;
-    }
```

Comment:
I think the idea here was to check the consistency of the LRU head pointers and the flags of the entries. Can we preserve/extend this? Also, can we add some `ASSERT`s that make sure that no item has both `InHighPriPool` and `InLowPriPool` set?

Reply:
Done, thanks.